### PR TITLE
House price prediction model completed. Closes [#597]

### LINKED ISSUE #597: [Add Content]: Mini ML Project to predict the house price
I propose to make ML project using TensorFlow or Scikit-Learn (as per your preference) to predict house prices using the house price dataset.

This project will be aimed to clear the concepts of classification and regression in machine learning.

Kindly assign me this issue under the label GSSoC'24

thanks and regards

### PR DESCRIPTION
**Pull request by seeratfatima19**
## PR Description

House price prediction model completed and jupyter notebook file is added containing the visualization, preprocessing and classification of data.

## Related Issues

- Closes #597 

### Checklist

- [X ] I have gone through the [contributing guide](https://github.com/animator/learn-python/blob/main/CONTRIBUTING.md)
- [ X] I have updated my branch and synced it with project `main` branch before making this PR

## Undertaking

I declare that: 

1. The content I am submitting is original and has not been plagiarized.  
2. No portion of the work has been copied from any other source without proper attribution.  
3. The work has been checked for plagiarism, and I assure its authenticity.  

I understand that any violation of this undertaking may have legal consequences that I will bear and could result in the withdrawal of any recognition associated with the work. 

- [X ] I Agree

### CODE REVIEW

**Comment by animator** [comment]
Read the contributing guide. Only markdown files are accepted.

**Comment by seeratfatima19** [comment]
okay I'll put all code in markdown file
